internal/pkg/cli/pipeline_init.go

```diff
@@ -178,44 +184,26 @@ func (o *initPipelineOpts) Ask() error {
 
 // Execute writes the pipeline manifest file.
 func (o *initPipelineOpts) Execute() error {
-	secretName := o.secretName()
-	_, err := o.secretsmanager.CreateSecret(secretName, o.githubAccessToken)
-
-	if err != nil {
-		var existsErr *secretsmanager.ErrSecretAlreadyExists
-		if !errors.As(err, &existsErr) {
+	if o.provider == ghProviderName {
+		if err := o.storeGitHubAccessToken(); err != nil {
 			return err
 		}
-		log.Successf("Secret already exists for %s! Do nothing.\n", color.HighlightUserInput(o.githubRepo))
-	} else {
-		log.Successf("Created the secret %s for pipeline source stage!\n", color.HighlightUserInput(secretName))
-	}
-	o.secret = secretName
-
-	var envConfigs []*config.Environment
-	for _, environment := range o.environments {
-		envConfig, err := o.store.GetEnvironment(o.appName, environment)
-		if err != nil {
-			return fmt.Errorf("get config of environment: %w", err)
-		}
-		envConfigs = append(envConfigs, envConfig)
 	}
 
 	// write pipeline.yml file, populate with:
-	//   - github repo as source
+	//   - git repo as source
 	//   - stage names (environments)
 	//   - enable/disable transition to prod envs
-
-	err = o.createPipelineManifest(envConfigs)
+	var err error
+	err = o.createPipelineManifest()
 	if err != nil {
```

Comment:
nit: a slightly nice way of writing this in go is like this: 
```suggestion
	if err := o.createPipelineManifest(); err != nil {
```
This way the `err` variable only exists within the scope of the if-block

Reply:
done